asn1acn.cpp

```diff
@@ -109,6 +119,29 @@ ExtensionSystem::IPlugin::ShutdownFlag Asn1AcnPlugin::aboutToShutdown()
     return SynchronousShutdown;
 }
 
+void Asn1AcnPlugin::loadSettings()
```

Comment:
Serviceprovider will be using QSettings, but this is important  to load them during plugin initialization. We do not know when the user will request to change those settings via tools menu, and we need to know what fallbacks  to be display by then.